ui/admin/tests/acceptance/auth-methods/oidc-test.js

```diff
@@ -67,39 +72,55 @@ module('Acceptance | auth-methods | oidc', function (hooks) {
 
   test('can view oidc state', async function (assert) {
     await visit(urls.authMethod);
-    await click('.rose-layout-page-actions .rose-dropdown-trigger');
+    await click(CHANGE_STATE_SELECTOR);
```

Comment:
note: I didn't use our new pattern commonSelectors since we have a separate ticket for that